v22.1/refresh.md

```diff
@@ -15,20 +15,19 @@ CockroachDB does not support materialized views that are refreshed on [transacti
 
 The user must be the [owner](owner-to.html) of the materialized view or have [admin](security-reference/authorization.html#admin-role) privileges.
 
-## Syntax
+## Synopsis
 
-~~~
-REFRESH MATERIALIZED VIEW [CONCURRENTLY] view_name [WITH [NO] DATA]
-~~~
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/release-{{ page.version.version | replace: "v", "" }}/grammar_svg/refresh_materialized_views.html %}
+</div>
 
 ## Parameters
 
  Parameter | Description
 -----------|-------------
-`CONCURRENTLY` | (*Default behavior*) This keyword is a no-op, added for PostgreSQL compatibility. All materialized views are refreshed concurrently with other jobs.
+`opt_concurrently` | `CONCURRENTLY` (*Default behavior*) This keyword is a no-op, added for PostgreSQL compatibility. All materialized views are refreshed concurrently with other jobs.
```

Comment:
I know that "no-op" is a commonly used phrase, but do you think it bears spelling out the meaning for a general reader?

Reply:
Also, should we not italicize "Default behavior" per our style usage?